templates/mattermost/0/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+services:
+  mattermost:
+    image: thanhpv102/mattermost-team
```

Comment:
Don't use generic docker image tag, implicit neither explicit, they are not reproducible. Please use concrete docker image version. 
```
-    image: thanhpv102/mattermost-team
+    image: thanhpv102/mattermost-team:4.3.0
```